tv_apps/smart-home/js/home.js

```diff
@@ -382,6 +382,8 @@
       } else {
         scrollable.insertNodeOver(newCardElem, idx);
       }
+
+      this.addFolderButton.classList.remove('hidden');
```

Comment:
I thought it will be followed by a `card-removed` event if it is a folder inserted. Nevertheless, I'll add it just in case.